pages/getting-started/installation-guide/index.md

```
@@ -51,16 +51,17 @@ With your favorite browser, visit the URL `http://localhost:8000`
+Run Vagrant from the official [Vagrant Build](https://jereme.dev/bludit-vagrant).
 
 ```
-$ git clone https://github.com/mhancoc7/Bludit-Vagrant.git
+$ git clone https://github.com/jeremehancock/Bludit-Vagrant.git
 $ cd Bludit-Vagrant
 $ vagrant up
 ```
 
 With your favorite browser, visit the URL `http://localhost:8080`
 
+Get more info [here](https://jereme.dev/bludit-vagrant).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)